id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoDraftServiceImpl.java

```diff
@@ -365,6 +365,7 @@ public IdResponseDTO publishDraft(String regId) throws IdRepoAppException {
 				anonymousProfileHelper.buildAndsaveProfile(true);
 				publishDocuments(draft, uinObject);
 				this.discardDraft(regId);
+				proxyService.sendGenericIdentityEvents(uin,true,regId);
```

Comment:
With the latest design there is no generic identity events, we have remove the events and related all code